### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by Paul-Clue**
# What I did:
- I created a webpage made almost entirely of Javascript.
- The information on the page changes when any navigation link is pressed. The information changes without the page reloading.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+// import _ from 'lodash';
+import './style.css';
+import nav from './navigation';
+import home from './home';
+import contact from './contact';
+import about from './about';
+
+const app = document.querySelector('#content');
+app.setAttribute('style', 'width: 100%; height: 100%;');
+// const app2 = document.querySelector('#content');
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!!! :muscle:

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// console.log("Is everything working?");
+// import _ from 'lodash';
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!!! :muscle:

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+// hme.addEventListener('click', () => {
+
+//   console.log(app.childNodes);
+//   const head = document.querySelector('h1');
+//   const para = document.querySelector('p');
+//   app2.removeChild(head);
+//   app2.removeChild(para);
+//   app.appendChild(contact());
+
+// });
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!!! :muscle:

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  if (value === 'home') {
+    value2 = home();
+  } else if (value === 'contact') {
+    value2 = contact();
+  } else {
+    value2 = about();
+  }
+
+  app.removeChild(app.childNodes[1]);
+  // app.removeChild(para);
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!!! :muscle:

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#content {
+  /* background: url('./food.jpg'); */
```
Comment (addod19) [inline src/style.css]: Kindly remove commented code, use git history instead!!! :muscle:

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import contact from './contact';
+import about from './about';
+
+const app = document.querySelector('#content');
+app.setAttribute('style', 'width: 100%; height: 100%;');
+// const app2 = document.querySelector('#content');
+app.appendChild(nav());
+app.appendChild(home());
+
+function change() {
```
Comment (addod19) [inline src/index.js]: Let us kindly use the ES6 arrow syntax instead!!!

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Paul-Clue, great job with the project structure :100: The is a feature for a tabbed menu :smile: which avoids page reload :+1: Content of each tabbed is stored in a module, each module exports a function that creates a `div` element adds appropriate content and style and appends to the DOM element. Awesome. We are almost done. Kindly check the comments below:

- Let us remove all commented piece of code in our application
- Let us add commands like build, development in the package.json inside `scripts'. So we can run those script commands
- Let us kindly use the ES6 arrow syntax instead of the `function` keyword

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: package.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (afanimartin) [inline package.json]: - Update your `scripts` section accordingly
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
}
```

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes. 
### Change(s) made 🌵 🌵 🌵 
- Use of ES6 syntax
- Comments removed
### Change(s) to make 🙅🏿 🙅🏿 🙅🏿 
- Add necessary commands to the `scripts` section of `package.json` file. Check comment.

### Martin Afani
#### Slack: whiz25

File: package.json (hunk 7 above)
Comment (Paul-Clue) [reply package.json]: Thank you. I have not seen those commands in any of the lessons.

**Review by afanimartin — APPROVED** [review]
## Approved 👏🏿 👏🏿 👏🏿 
Necessary commands added to `package.json` file. Merge and proceed to the next project.

### Martin Afani
#### Slack: whiz25

**Pull request merged**